src/OpenTelemetry.Exporter.InMemory/InMemoryExporterMetricsExtensions.cs

```diff
@@ -147,11 +147,13 @@ public static MeterProviderBuilder AddInMemoryExporter(
         });
     }
 
-    private static MetricReader BuildInMemoryExporterMetricReader(
+    private static PeriodicExportingMetricReader BuildInMemoryExporterMetricReader(
```

Comment:
This was an unexpected change, so I'm making a note here for other reviewers...
Visual Studio IntelliSense is giving this message:

> CA1859 Change return type of method 'BuildInMemoryExporterMetricReader' from 'OpenTelemetry.Metrics.MetricReader' to 'OpenTelemetry.Metrics.PeriodicExportingMetricReader' for improved performance